### PR TITLE
Update SlideOutView doccoments

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

**Review by TikhonovaTatyana — APPROVED** [review]

File: js/ui/slide_out_view.js [hunk 1]
```diff
@@ -82,15 +82,17 @@ var SlideOutView = Widget.inherit({
             /**
             * @name dxSlideOutViewOptions_menuTemplate
             * @publicName menuTemplate
-            * @type template
+            * @type_function_param1 menuElement:dxElement
+            * @type template|function
```
Comment (San4es) [inline js/ui/slide_out_view.js:86]: The "template" type already includes "function":
https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/template/

Comment (DokaRus) [reply js/ui/slide_out_view.js:86]: We need this to generate the `Function parameters:` block like [there](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxCalendar/Configuration/#cellTemplate)

Comment (San4es) [reply js/ui/slide_out_view.js:86]: @lukyanovas @IlyaKhD 

Can we do this without adding "|function" to `@type`?

Comment (lukyanovas) [reply js/ui/slide_out_view.js:86]: @San4es No. Now it is necessary for correct function detection

**Review by lukyanovas — APPROVED** [review]

**Pull request merged**